go.mod

```diff
@@ -46,5 +47,6 @@ require (
 
 replace (
 	github.com/ipfs/go-ipfs => github.com/lazyledger/go-ipfs v0.8.0-lazypatch
+	github.com/ipfs/go-verifcid => github.com/lazyledger/go-verifcid v0.0.1-lazypatch
```

Comment:
Maybe add a comment here. The reason is basically that the ipld plugin (which uses go-verifcid) will be used in the top-level module (lazyledger-core). Apparently, without this line, the go-verifcid version gets overwritten by the root module.

Reply:
I forgot in what commit, you're right, there needs to be a comment there https://github.com/lazyledger/lazyledger-core/blob/c07235d14117e60b72626a9138c7fbcdc9468b7b/go.mod#L50